src/vmspawn/vmspawn.c

```diff
@@ -687,6 +688,30 @@ static int run_virtual_machine(void) {
         if (r < 0)
                 return log_oom();
 
+        if (use_kvm && kvm_device_fd > 0) {
+                accel = "kvm,device=/dev/fdset/1";
```

Comment:
hmm what's /dev/fdset/? That doesn't exist here. Is that a magic string for qemu?

Reply:
This tells qemu to receive the KVM device from the first fd set, we provide a fd to `/dev/kvm` in set one if we are provided one from `sd_listen_fds`, so this argument is required so that qemu uses it instead of trying to open `/dev/kvm` itself.

Reply:
can you add a comment that /dev/fdset/ is a magic string for qemu and not an actual fs path then

Reply:
Fixed in eae47ee9d9.